card.py

```diff
@@ -24,3 +24,7 @@ def value_name(self):
 
     def __repr__(self):
         return f'{self.value_name}{self.suit}'
+
+    def __hash__(self):
+        suit_dict = {'♣': 1, '♥': 2, '♠': 3, '♦': 4}
+        return self.value * 10 + suit_dict.get(self.suit)
```

Comment:
My money says you can do `return repr(self)`. Your repr method is returning a unique string for for each card. We can likely leverage that.

Reply:
`__hash__` apparently requires an int output.

Reply:
News to me. Then yea, this is probably fine. There is probably a better way to do it, I just can't think of it at the moment. I'll see what I can see and we can do something in a follow up PR if we find something.